frontend/src/router/index.ts

```diff
@@ -100,6 +101,25 @@ const routes = [
         component: GrantApplicationAdminView,
         beforeEnter: beforeEnterHandlers[routeItems.grantAppAdmin.name],
     },
+    {
+        path: routeItems.grantDelegatedAdmin.path,
+        name: routeItems.grantDelegatedAdmin.name,
+        meta: {
+            requiresAuth: true,
+            requiresAppSelected: true,
+            requiredPrivileges: [FAM_ADMIN_ROLE],
```

Comment:
this will be app admin, only app admin can grant delegated admin

Reply:
Thanks, I have changed it.